gtsfm/data_association/data_assoc.py

```diff
@@ -114,36 +114,45 @@ def run(
                 raise RuntimeError("Some cameras must have been dropped ")
             triangulated_data.add_camera(cam)
 
-        mean_3d_track_length, median_3d_track_length, track_lengths_3d = SfmResult(triangulated_data, None).get_track_length_statistics()
+        mean_3d_track_length, median_3d_track_length, track_lengths_3d = SfmResult(
+            triangulated_data, None
+        ).get_track_length_statistics()
 
         logger.debug("[Data association] output number of tracks: %s", num_accepted_tracks)
         logger.debug("[Data association] output avg. track length: %s", mean_3d_track_length)
 
         # dump the 3d point cloud before Bundle Adjustment for offline visualization
         points_3d = [list(triangulated_data.track(j).point3()) for j in range(num_accepted_tracks)]
+        # bin edges are halfway between each integer
+        histogram_track_lengths, _ = np.histogram(track_lengths_3d, bins=np.linspace(-0.5, 10.5, 12))
```

Comment:
`track_lengths_histogram`?

Reply:
Also, why bin it from -0.5. We can use the ` np.histogram(track_lengths_3d, bins=8, range=(2, 10))`